app/components/Calendar.tsx

```diff
@@ -1,6 +1,10 @@
+/* eslint-disable function-paren-newline */
```

Comment:
Why eslint disabled? If it is due to some configuration issue let us add a comment here and fix it.

Reply:
+1

Reply:
@DeRaowl @sumitd94 it is an issue of conflict between eslint and prettier with regards to the code formatting. We had a discussion regarding this in the team call and will be creating a separate issue to configure ESlint and Prettier together to address this issue.

Reply:
@harshith-venkatesh created an issue for the same. Link here: https://github.com/Real-Dev-Squad/website-calendar/issues/135